Resources/doc/index.rst

```diff
@@ -1,50 +1,51 @@
-
-
-LiipImagineBundle
-=================
```

Comment:
what happened here? is it line endings?

Reply:
it's like the original file, don't know why my commit pushed as a new file. The line 45 aws the only one i added.